index.js

```diff
@@ -22,10 +22,11 @@ function fixOrientation (url, opts, fn) {
   
   try { tags = EXIF.readFromBinaryFile(buf.buffer) } catch (err) {}
 
+  var supportedOrientation = [3,6,8];
+
   var toRotate = tags.Orientation
-    && typeof tags.Orientation == 'number'
-    && (tags.Orientation == 6
-    || tags.Orientation == 8);
+              && typeof tags.Orientation == 'number'
+              && supportedOrientation.includes(tags.Orientation);
```

Comment:
could you please undo the indentation changes?

Reply:
Fixed